desc/objectives/_geometry.py

```diff
@@ -567,6 +567,12 @@ class PlasmaVesselDistance(_Objective):
         Defaults to ``LinearGrid(M=eq.M_grid, N=eq.N_grid)``.
     use_softmin: bool, optional
         Use softmin or hard min.
+    use_signed_distance: bool, optional
```

Comment:
Is the signed algorithm much slower? Or is it common to use different grids for the two different surfaces? I'm wondering if we should just always use the signed version (or at least default to the signed version)

Reply:
Currently it is yes, as it has to loop over the zeta planes to assign the signs. I have not tested it extensively though but I think it should be optional for now